components/dex.libsonnet

```diff
@@ -7,13 +7,20 @@ local defaults = {
   image: error 'must provide image',
   version: error 'must provide version',
   namespace: error 'must provide namespace',
+  tlsSecret: '%s-tls' % [defaults.name],
+  tlsCertMapEntry: 'tls.crt',
```

Comment:
Can we still change the names of these fields?

Reply:
Ouch, sorry! Yes, absolutely. I focused so much on the rebase that I forgot this one :-)

Reply:
PR updated.